content/news/029/index.md

```diff
@@ -73,6 +73,30 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ## Tooling Updates
 
+### [Blackjack]
+
+![Animated gif showcasing the main features of Blackjack](blackjack.gif)
```

Comment:
Could you please write a more descriptive alt text? Something like "Demo: connecting visual nodes and tweaking various parameters to alter the model in real-time" etc